python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -100,18 +96,25 @@ def main(config_args):
                                                 device=device, train_task=True)
     else:
         dataloader = GSgnnNodeDataLoader(train_data, train_data.train_idxs, fanout=config.fanout,
-                                         batch_size=config.batch_size, device=device,
-                                         train_task=True)
+                                         batch_size=config.batch_size,
+                                         device=device, train_task=True,
+                                         reconstructed_embed_ntype=config.reconstructed_embed_ntype)
     # we don't need fanout for full-graph inference
     fanout = config.eval_fanout if config.use_mini_batch_infer else []
+    val_dataloader = None
+    test_dataloader = None
     if len(train_data.val_idxs) > 0:
+        _ntype = config.reconstructed_embed_ntype
         val_dataloader = GSgnnNodeDataLoader(train_data, train_data.val_idxs, fanout=fanout,
                                              batch_size=config.eval_batch_size,
-                                             device=device, train_task=False)
+                                             device=device, train_task=False,
+                                             reconstructed_embed_ntype=_ntype)
```

Comment:
Why the implementation of L101 and L111 is different?

Reply:
what do you mean?